src/App.vue

```diff
@@ -134,6 +148,10 @@ export default class App extends Mixins(mixins.TransactionMixin, NodeErrorMixin)
     return this.$route.name === PageNames.About;
   }
 
+  get blockNumberFormatted(): string {
+    return this.blockNumber.toLocaleString();
```

Comment:
After language switching, this formatted value is not changing. I guess it's possible to add `language` getter here and use it :
`this.blockNumber.toLocaleString(this.language)`

Reply:
it's better to format with FPNumber - just for consistency